src/engine/controls/cuecontrol.cpp

```diff
@@ -70,6 +70,25 @@ inline int hotcueNumberToHotcueIndex(int hotcueNumber) {
     }
 }
 
+void appendCueHint(HintVector* pHintList, const mixxx::audio::FramePos& frame, Hint::Type type) {
+    VERIFY_OR_DEBUG_ASSERT(pHintList) {
+        return;
+    }
+
+    if (frame.isValid()) {
+        const Hint cueHint = {
+                .frame = static_cast<SINT>(frame.toLowerFrameBoundary().value()),
+                .frameCount = Hint::kFrameCountForward,
+                .type = type};
```

Comment:
I thought this was just a slightly different syntax, but apparently its a C++20 feature (which we don't support yet). They normal `key: value` syntax I proposed is actually C++17 compatible.

Reply:
I have now surrounded them with comments. That should build even with MSVC .. hopefully.

Reply:
Yeah, a plain initializer list should be no problem. But list initialization with designated initializers is C++20...